packages/chart/src/components/SparkLine.jsx

```diff
@@ -96,6 +96,7 @@ export default function SparkLine({ width: parentWidth, height: parentHeight })
 
       yScale = config.runtime.xAxis.type === 'date' ? scaleLinear({ domain: [Math.min(...xAxisDataMapped), Math.max(...xAxisDataMapped)] }) : scalePoint({ domain: xAxisDataMapped, padding: 0.5 })
 
+      // eslint-disable-next-line
```

Comment:
question: im not seeing any lint errors when removing this disable line

Reply:
removed